airbyte-integrations/connector-templates/source-java-jdbc/README.md

```diff
@@ -0,0 +1,68 @@
+# Source {{titleCase name}}
+
```

Comment:
Huh did we forget to add this previously?

Reply:
yes